integration/test_helpers/helpers.go

```diff
@@ -47,7 +43,7 @@ func RunCommand(cmd string, args ...string) *Session {
 
 	session, err := Start(command, GinkgoWriter, GinkgoWriter)
 	Expect(err).NotTo(HaveOccurred())
-	session.Wait()
+	session.Wait(10 * time.Second)
```

Comment:
Let's keep all waits in constant at top of file since I want these to be as small as possible (i.e., need tests to be as fast as possible) and making sure we can change these easily to try smaller values